wordpress-mobile/WordPress-iOS
# Reader: Add a footer view for the Reader tags feed

Fixes #23108

## Description

Adds the "More" footer view to the Reader tags feed.

Note: There's an existing issue when navigating to a tag feed from the "Your Tags" feed. This is due to the following chain of events:
1. A reader stream view controller is created with the tag slug
2. A lookup happens and then the `readerTopic` of the stream view controller gets set
3. The `didSet` of the `readerTopic` sets the `inUse` property on the tag managed object and the context is saved
4. The existing "Your Tags" feed now sees a change to the tag object
5. That tag cell is reloaded

This causes the collection view offset to briefly show at the beginning, and then it'll update back to the end of the content when navigating back to the "Your Tags" feed from the tag feed.

I couldn't find a good way to fix this because it appears the `inUse` property is currently used. I'll keep attempting to look for a solution.

## Screenshots

| Light | Dark |
|---|---|
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-26 at 17 18 11](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/085e4553-2d7f-4b86-be9b-e4a8ba0e2ccf) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-26 at 17 18 23](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/73a7ec6e-5e40-46e9-b237-b38f76806af3) |

## Testing

To test:
- Launch Jetpack and login
- Navigate to the "Your Tags" feed
- Scroll to the end of a tag's collection view
- 🔎 **Verify** the "More" footer view is styled correctly
- Tap on the view
- 🔎 **Verify** it navigates to that tag's feed

## Regression Notes
1. Potential unintended areas of impact
Should be none.

7. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

8. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:


Testing checklist:
